pkg/pillar/kubeapi/nokube.go

```diff
@@ -6,15 +6,16 @@
 package kubeapi
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/lf-edge/eve/pkg/pillar/base"
 	"github.com/lf-edge/eve/pkg/pillar/pubsub"
 )
 
 // WaitForKubernetes in this file is just stub for non-kubevirt hypervisors.
-func WaitForKubernetes(
-	string, *pubsub.PubSub, *time.Ticker) error {
+func WaitForKubernetes(string, *pubsub.PubSub, *time.Ticker,
+	...pubsub.ChannelWatch) error {
```

Comment:
This change does not belong to this PR.

Reply:
My mistake @milan-zededa, I'll remove it